packages/ui/src/contexts/MultiProxyContext.tsx

```diff
@@ -225,6 +226,7 @@ const MultiProxyContextProvider = ({ children }: MultisigContextProps) => {
   const watchedAddressesIds = useAccountId(watchedAddresses)
   const {
     isLoading: isMultisigsubLoading,
+    isFetchingMultisigs,
```

Comment:
Beside my previous comment regarding `isLoading`, this is not the only place where things could be loading.
I'll admit this context is way too complex. The reason is query optimisations that obliged us to have actually 2 subscriptions. 1 for the multisigs + proxy, and 1 for the watched pure proxies. 

I think we should use the `isLoading` status and remove this one, but just FYI the other subscription in this file `usePureByIdsSubscription` is also mutating the `multiProxyList` and this is why relying only on this subscription wouldn't work well in some scenario.

Reply:
Did the behavior feel weird/inconsistent when using isLoading? The isLoading state that we export outside of the context is much more than just a multisig subscription. Also f there is a new account or watched account.re-do the subscription (it's part of the id of the hook) and a whole refetch is happening. If nothing is changing, then because it's a subscription, there's actually no refetching.

Reply:
Changed back to "isLoading" state